manifests/centosimage.pp

```diff
@@ -29,18 +30,32 @@
     before  => Docker::Image['centosagent']
   }
 
+  # Mount the gem cache directory because docker doesn't follow symlinks
+  # mount --bind /var/cache/rubygems rubygems
+  file {'/var/docker/centosagent/rubygems':
+    ensure => directory,
+    before => Mount['/var/docker/centosagent/rubygems'],
+  }
+
+  mount {'/var/docker/centosagent/rubygems':
```

Comment:
@samuelson Yeah, just need it to be visible inside the container, so at start time is fine. I can remove both of those mounts and update the code that you referred to above.